### DISCUSSION TITLE
TypeError: __init__() got an unexpected keyword argument 'ml'

### DISCUSSION THREAD

**Discussion by prateekgml** [microsoft/FLAML#581] [Q&A]
Hi,

I am trying to fit the model with the ensemble option-

`automl.fit(X_train=X_train, y_train=y_train, task='classification', ensemble=True, use_ray=True, seed=42)`

But I am getting the following error:-

![image](https://user-images.githubusercontent.com/97145738/173089260-535f9326-3ca3-4652-b855-ac7e0cb6396e.png)

Before this error followings info is printed in console:-

![image](https://user-images.githubusercontent.com/97145738/173089450-9c1781a3-4c76-4f47-889e-96f1bd717630.png)

Environment used is Kaggle and version of FLAML is 1.0.6.

My notebook is-
[amex-perquet.txt](https://github.com/microsoft/FLAML/files/8879788/amex-perquet.txt)

**Comment by sonichi** [comment]
I created an issue and a PR to fix it #583

**Reply by prateekgml** [reply]
Thanks @sonichi for this PR.

**Reply by sonichi** [reply]
The PR has been merged. You can test it by installing from git.

**Reply by prateekgml** [reply]
Can I test it by this command- `!pip install flaml --upgrade`

**Reply by sonichi** [reply]
`!pip install git+https://github.com/microsoft/FLAML.git`

**Reply by prateekgml** [reply]
I have run the Kaggle notebook after installing the latest from the git and it's in progress. I will update once notebook is completed.

**Reply by prateekgml** [reply]
The notebook run is completed and the last logs in the server console are as below-

![image](https://user-images.githubusercontent.com/97145738/173217175-61236e4d-5795-4cd1-9b6b-233596a36b3d.png)

It seems this error has gone. Thanks for the fix @sonichi
